RELEASING.adoc

```diff
@@ -1,80 +1,210 @@
 
-The steps to perform a release are relatively simple thanks to the way link:.travis.yml[Travis] is setup.
+== Creating a new JIRA issue
 
-Before running these steps, it is assumed the link:https://github.com/kiali/kiali/tree/master[kiali master] branch and the link:https://github.com/kiali/kiali-ui/tree/master[kiali-ui master] branch are all up to date and include all code that is to be released and that your local `master` branches are up to date with the remote `master` branches.
+Create a new issue on link:https://issues.jboss.org/browse/KIALI[JIRA] detailing the
+release, and add sub-tasks if necessary.
 
-For the steps below, it is assumed the image version being released is v1.0.0. Obviously, this version string for the docker image will change as new images are released.
+== Updating Docs
 
-== Kiali-UI
+First step is to make sure the "Getting Started" document for the website is up
+to date and works with the new version.
 
-1) Tag the Kiali-UI master branch in this repo: https://github.com/kiali/kiali-ui
+The docs can be found link:https://github.com/kiali/kiali.io[here]. Do the
+changes, and create a PR.
 
-NOTE: You must have write permissions to the public GitHub repository in order to be able to do this.
+== Updating Kiali UI
 
-For example,
+The specifics of releasing Kiali UI can be found
+link:https://github.com/kiali/kiali-ui/blob/master/RELEASING.adoc[here].
 
-```
-$ git tag v1.0.0
-$ git push origin v1.0.0
-```
+Releasing Kiali UI should come before releasing the core, as the image is
+created as soon as the tag for core is pushed to master.
+
+== Publish API docs
+
+API docs are generated by swagger, and this comment will both install it and
+generate the updated documentation.
 
-Note that this is the same version string that will also tag the kiali repo - this is to ensure they are in-sync and can be used to link the kiali and kiali-ui repo at the time of the release. In other words, this is not the kiali-ui version string that we are getting in the next step.
+[source, bash]
+----
+$ make swagger-install swagger-gen
+----
 
-2) Go to https://www.npmjs.com/package/@kiali/kiali-ui and find the version of the UI that should be built for this release. We will put this console version number in Kiali's Makefile - see below.
+== Update Makefile
 
-_Finally, it is still undetermined how future releases are going to be done with respect to kiali-ui and its own version string. It is possible kiali-ui and kiali repos will be combined into one and thus both kiali and the kiali-ui version strings will be united as a single version string. Today, the version string of kiali-ui as it is seen in the NPM repository is determined within the kiali-ui travis configuration file. This may change in the future. But for now, we need to go to the NPM repository to get the version string of kiali-ui that is to be pulled into the kiali image._
+Change the VERSION variable on Makefile to the new version, this is what is
```

Comment:
Change the `VERSION` variable in the Makefile to the new version.  This version will be displayed in the UI, and is also what Travis will use to trigger the release process.